pages/_app.tsx

```diff
@@ -70,6 +70,12 @@ declare global {
 const Analytics = () => {
   return (
     <>
+      {/* Crazy Egg interaction data */}
+      <script
```

Comment:
Just a note, these need to be added as <Script vs <script.. the JS is a little different. Look at the other examples

Reply:
See https://github.com/gravitational/docs/pull/431 as an example.